examples/asr/speech_to_text.py

```diff
@@ -69,8 +70,18 @@ def main(cfg):
     exp_manager(trainer, cfg.get("exp_manager", None))
     asr_model = EncDecCTCModel(cfg=cfg.model, trainer=trainer)
 
+    if cfg.model.get("load_weights_from_checkpoint", None):
+        logging.info(f"Loading checkpoint '{cfg.model.load_weights_from_checkpoint}' ...")
+        asr_model.update_weights(checkpoint_path=cfg.model.load_weights_from_checkpoint)
+
     trainer.fit(asr_model)
 
+    if hasattr(cfg.model, 'test_ds') and cfg.model.test_ds.manifest_filepath is not None:
+        gpu = 1 if cfg.trainer.gpus != 0 else 0
+        trainer = pl.Trainer(gpus=gpu)
```

Comment:
You also need to add precision + AMP backend flags. So the flags `gpus`, `precision`, `amp_level` and `amp_backend` all need to be copied from cfg with cfg.get('X', <default if X not in cfg>).

Reply:
Done

Reply:
done